changelog/upgrade-notes.md

```diff
@@ -31,6 +31,10 @@ but with additional css:
 }
 ```
 
+#### Removal of `ol.source.TileVector`
+
+The experimental `ol.source.TileVector` class has been removed. Its functionality is now provided by `ol.source.VectorTile`. The API is the same, except for the `tileLoadFunction`, which is no longer an `ol.TileVectorLoadFunctionType`, but an `ol.TileLoadFunctionType`.
+
```

Comment:
One obvious reason could be that everywhere else (mapbox and related) has named it VectorTiles and I could see it make sense that a Tile of VectorTiles become a VectorTile. 

I am mostly thinking that new developers will google for VectorTile(s) and not TileVector when looking for related resources and I have no idea if google is smart enough to see that its the same thing. 

I also see why one would keep the original name if its been exposed to many of the current developers. I only speak as an outsider/consumer of ol3 here, so use the input if it's relevant.